integration-tests/setEnvVars.js

```diff
@@ -12,3 +12,5 @@ process.env.STATE_MACHINE_ROLE =
 
 process.env.NINO_ATTEMPTS_TABLE = "pdv-matching-nino-attempts";
 process.env.NINO_USERS_TABLE = "pdv-matching-nino-users";
+
+process.env.REDIRECT_URI = "https://cri.core.stubs.account.gov.uk/callback";
```

Comment:
this is in the wrong place, but take note of my previous comment not defining a separate const for REDIRECT but deriving from const base on a CLIENT_URL